test

```diff
@@ -233,6 +233,18 @@ function fmt_pass {
 		exit 255
 	fi
 
+	# TODO: check other markdown files when marker handles headers with '[]'
+	if which marker >/dev/null; then
+		echo "Checking marker to find broken links..."
+		markerResult=`marker --root ./Documentation 2>&1 || true`
```

Comment:
You might want to use `--skip-http`. Otherwise, marker will do a bunch of HEAD and GET requests to check remote resources. This is pretty slow and might spuriously fail.